app/controllers/games_controller.rb

```diff
@@ -25,12 +30,22 @@ def authorize
   end
 
   def set_game
-    @game = Game.joins(:srdc).find_by(speedrun_dot_com_games: {shortname: params[:game]}) || Game.find_by(id: params[:game])
+    @game = Game.joins(:srdc).find_by(speedrun_dot_com_games: {shortname: params[:game]}) ||
+            Game.find_by(id: params[:game])
+  end
 
+  def redirect_if_not_found
```

Comment:
Instead of creating a new method for this we should probably be consistent with other controller where it uses `Game.find` and then `rescue`s from the method if the record doesn't exist.